app.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+require 'bundler'
+
+Bundler.require
+
+require_relative 'restaurant'
+require_relative 'foodbot'
+
+Dir["restaurants/*.rb"].each {|file| require_relative file }
+
+restaurants = Restaurant.descendants.map(&:new).map {|r| [r.command_name, r] }.to_h
+foodbot = Foodbot.new(restaurants)
+
+get "/" do
```

Comment:
Oops, was easier to test :)
On 17 May 2015 20:51, "radiokills" <notifications@github.com> wrote:

> In app.rb
> <https://github.com/radiokills/foodbot/pull/2#discussion_r30473316>:
>
> > @@ -0,0 +1,17 @@
> > +require 'rubygems'
> > +require 'bundler'
> > +
> > +Bundler.require
> > +
> > +require_relative 'restaurant'
> > +require_relative 'foodbot'
> > +
> > +Dir["restaurants/*.rb"].each {|file| require_relative file }
> > +
> > +restaurants = Restaurant.descendants.map(&:new).map {|r| [r.command_name, r] }.to_h
> > +foodbot = Foodbot.new(restaurants)
> > +
> > +get "/" do
>
> :'(
>
> y u get? y u no post?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/radiokills/foodbot/pull/2/files#r30473316>.
>